build/downloader/src/index.ts

```diff
@@ -0,0 +1,93 @@
+import os from 'os'
+import path from 'path'
+import yargs from 'yargs'
+import { cleanFiles, initProject, installFile, listFiles, useFile } from './cli'
+import { getAppDataPath } from './utils'
+
+export interface CommonArgs {
+  appData: string
+  platform: string
+  output: string
+}
+
+const getCommonArgv = (argv): CommonArgs => {
```

Comment:
I've had some issues with yargs and typescript... Basically, I want to have appData / platform / output available for each commands, but I don't want to duplicate those 3 entries on each command... And since `argv` is inferred from options passed directly to that specific command, all arguments ends up as "unknown" and would need to be cast as a string... 

My approach is not ideal, but I couldn't find a better alternative